### ISSUE TITLE
Wont play videos or thumbnails

### ISSUE THREAD

**Issue by blade10101** [lamarios/clipious#495]
When connected to my instance. No thumbnails or videos play

**Comment by lamarios** [comment]
https://github.com/lamarios/clipious/wiki/Common-Issues

**Comment by Acid4t** [comment]
use play integrity fix  that worked for me unsurprisingly

**Comment by blade10101** [comment]
I can't root my galaxy s24 phone to get this.

Feb 29, 2024 11:47:17 PM Acid4t ***@***.***>:

**Comment by blade10101** [comment]
I do have the domain set to https://invidious.darkness.services and port 443 as it's behind a reverse proxy. Freetube connects to it and plays just fine.

Feb 29, 2024 7:23:47 PM Paul Fauchon ***@***.***>:

**Comment by lamarios** [comment]
Did you check the wiki page I linked. What does the api show for thumbnails url.

**Comment by blade10101** [comment]
Yeah I went to the URL but it seemed to just mention. The external port and domain

Mar 1, 2024 7:45:32 AM Paul Fauchon ***@***.***>:

**Comment by blade10101** [comment]
Well I did notice it is showing http:// so I will set it to use https all the time.

Mar 1, 2024 7:45:32 AM Paul Fauchon ***@***.***>:
